File: 240513/씨 오 더블유 2/c-o-w-2.java
import java.util.*;
public class Main {
    public static void main(String[] args) {
        // 여기에 코드를 작성해주세요.
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();
        sc.nextLine();
        String str = sc.nextLine();
        char[] arr = str.toCharArray();
        int count = 0;

        for(int i=0;i<n-2;i++) {
            if(arr[i] == 'C') {
                for(int j=i+1;j<n-1;j++) {
                    if(arr[j] == 'O') {
                        for(int k=j+1;k<n;k++) {
                            if(arr[k] == 'W')
                                count++;
                        }
                    }
                }
            }
        }
        System.out.println(count);
    }
}

File: 240506/씨 오 더블유 2/c-o-w-2.kt
import java.util.Scanner

fun main() {
    val sc = Scanner(System.`in`)

    val n = sc.nextInt()
    val str = sc.next()
    var result = 0

    for (i in 0 until n) {
        for (j in i + 1 until n) {
            for (k in j + 1 until n) {
                if (str[i] == 'C' && str[j] == 'O' && str[k] == 'W') result++
            }
        }
    }

    print(result)
}